### PR TITLE
180890663/implement payment type property

### PR DESCRIPTION
**Pull request by LBHCallumM**
## Link to JIRA ticket

[Ticket](https://www.pivotaltracker.com/story/show/180890663)

## Describe this PR

Add PaymentType property to WorkOrder and JobStatusUpdate entities.

Please also review the data migration to see if I havent considered anything.
- Any workOrder that isnt overtime will be set to 'bonus'. No workOrders will be 'CloseToBase' after the migration.
- There is a risk that there will be a delay between the migration and frontend sending the payment_type field in each request. If this does happen, I may need to manually run this migration again.
- Migration Down will update is_overtime to true if payment_type is overtime, otherwise false. This migration does assume that the is_archived field is fully up to date.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### CODE REVIEW

File: RepairsApi/V2/Services/RepairsSnsService.cs [hunk 1]
```diff
@@ -67,7 +73,7 @@ private async Task<List<EntityEventSns>> CreateMessages(WorkOrder workOrder, IEn
             {
                 var op = await _operativesGateway.GetByIdAsync(operative.OperativeId);
 
-                msgs.Add(new EntityEventSns
+                var snsMessage = new EntityEventSns
```
Comment (Asura5) [inline RepairsApi/V2/Services/RepairsSnsService.cs:76]: Why is this necessary?

**Review by Asura5 — APPROVED** [review]

**Comment by Asura5** [comment]
Ensure the changes to this and the listener are merged at the same time

File: RepairsApi/V2/Services/RepairsSnsService.cs (hunk 1 above)
Comment (LBHCallumM) [reply RepairsApi/V2/Services/RepairsSnsService.cs:76]: It isn't. I added a feature toggle, but later removed it.

**Review by teodor-hristoforov-hackney — APPROVED** [review]
